CMake/fletch-tarballs.cmake

```diff
@@ -428,24 +428,23 @@ list(APPEND fletch_external_sources CppDB)
 
 # VTK
 if (fletch_ENABLE_VTK OR fletch_ENABLE_ALL_PACKAGES)
-  set(VTK_SELECT_VERSION 8.2 CACHE STRING "Select the version of VTK to build.")
-  set_property(CACHE VTK_SELECT_VERSION PROPERTY STRINGS 6.2 8.0 8.2)
+  set(VTK_SELECT_VERSION 9.0 CACHE STRING "Select the version of VTK to build.")
+  set_property(CACHE VTK_SELECT_VERSION PROPERTY STRINGS 8.0 8.2 9.0)
 endif()
 
-if (VTK_SELECT_VERSION VERSION_EQUAL 8.2)
+if (VTK_SELECT_VERSION VERSION_EQUAL 9.0)
+  set(VTK_version 9.0.1)
+  set(VTK_md5 "b3ba14d616c3b23583c42cffb585deac")
+elseif (VTK_SELECT_VERSION VERSION_EQUAL 8.2)
   set(VTK_version 8.2.0)
-  set(VTK_md5 "94ba8959b56dcfa6bac996158669ac36")
+  set(VTK_md5 "8af3307da0fc2ef8cafe4a312b821111")
```

Comment:
Why did the hash change for 8.2?  Was what we had before not the official 8.2 but only a pre-release?  I vaguely remember that needed to get 8.2 in before it was official.

Reply:
It looks like we switched from `.zip` to `.tar.gz`? That would obviously change the checksum...

Reply:
Correct, the checksums were updated. I switched because the download for 9.0.1 only presented a .tar.gz though I guess a zip was around somewhere if I looked.